test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs

```diff
@@ -0,0 +1,122 @@
+    {
+        public RavenDB_16311(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task CanTestOlapEtlScript()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var baseline = new DateTime(2020, 1, 1);
+
+                using (var session = store.OpenAsyncSession())
+                {
+                    for (int i = 0; i < 31; i++)
+                    {
+                        await session.StoreAsync(new Query.Order
+                        {
+                            Id = $"orders/{i}",
+                            OrderedAt = baseline.AddDays(i),
+                            ShipVia = $"shippers/{i}",
+                            Company = $"companies/{i}"
+                        });
+                    }
+
+                    for (int i = 0; i < 28; i++)
+                    {
+                        await session.StoreAsync(new Query.Order
+                        {
+                            Id = $"orders/{i + 31}",
+                            OrderedAt = baseline.AddMonths(1).AddDays(i),
+                            ShipVia = $"shippers/{i + 31}",
+                            Company = $"companies/{i + 31}"
+                        });
+                    }
+
+                    await session.SaveChangesAsync();
+                }
+
+                var database = await GetDatabase(store.Database);
+
+                using (database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+                {
+                    var result = (OlapEtlTestScriptResult)OlapEtl.TestScript(new TestOlapEtlScript
+                    {
+                        DocumentId = "orders/1",
+                        Configuration = new OlapEtlConfiguration()
+                        {
+                            Name = "simulate",
+                            Transforms =
+                                {
+                                    new Transformation()
+                                    {
+                                        Collections =
+                                        {
+                                            "Orders"
+                                        },
+                                        Name = "MonthlyOrders",
+                                        Script =
+                                            @"
+                                                var orderDate = new Date(this.OrderedAt);
+                                                var year = orderDate.getFullYear();
+                                                var month = orderDate.getMonth();
+                                                var key = new Date(year, month);
+
+                                                output('test output')
+
+                                                loadToOrders(partitionBy([['order_date', key]]),
```

Comment:
syntax was changed, need to remove the outer array i.e. `partitionBy(['order_date', key])`

Reply:
done